staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal/fields.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // EmptyFields represents a set with no paths
 // It looks like metav1.Fields{Raw: []byte("{}")}
-var EmptyFields metav1.FieldsV1 = func() metav1.FieldsV1 {
+var EmptyFields = func() metav1.FieldsV1 {
```

Comment:
I can't see a corresponding line in `.golint_failures`, are there many other errors in this package?

Reply:
Yes, this package is not contained in `.golint_failures`, but `verify-golint.sh` failed in this file and this is the only error in this package, so I fixed this also. 
I will separate this fix in different PRs.